d-sgov/d-sgov-vfr-1.8.0-model.ttl

```diff
@@ -1213,12 +1219,6 @@ d-sgov-vfr-1.8.0-pojem:kód-obce-v-lau
         rdfs:range          rdfs:Literal ;
         rdfs:subPropertyOf  <https://slovník.gov.cz/základní/pojem/vlastnost> .
 
-<http://slovník.gov.cz/datový/vfr-1.8.0/pojem/identifikátor-transakce-v-rúian-pro--městský-obvod/městskou-část>
-        a                   owl:DatatypeProperty , <https://slovník.gov.cz/základní/pojem/typ-vlastnosti> ;
-        rdfs:domain         <http://slovník.gov.cz/datový/vfr-1.8.0/pojem/městský-obvod/městská-část> ;
-        rdfs:range          rdfs:Literal ;
-        rdfs:subPropertyOf  <https://slovník.gov.cz/základní/pojem/vlastnost> .
-
 <http://slovník.gov.cz/datový/vfr-1.8.0/pojem/znak-městského-obvodu/městské-části>
```

Comment:
Lomítko určitě v identifikačním názvu pojmu nepoužívat. Bude způsobovat problémy ...

Reply:
ok, vidím, změním